index.js

```diff
@@ -1,3 +1,19 @@
-// javascript
+// Capture user's input on form submission
+let form = document.querySelector("form");
 
-alert("Hello from index.js!");
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+
+  console.log("Form submitted");
+});
```

Comment:
Nice - good use of an event listener and comment plan

Reply:
Good start. You are displaying alert messages successfully to the user if they submit invalid data.

It looks as though you are ready to start the [error handling](https://gedsargent.github.io/code-monster/guides/programming/harmony-music/error-handling/) tutorial on Code Monster 🚀